### PR TITLE
documentation imported from code.kx.com

### PR DESCRIPTION
**Pull request by StephenTaylor-Kx**
:warning: **Merge by 2023.03.01**

Documentation imported from code.kx.com before it is removed on 2023.03.01.

Markdown converted to GFM; links fixed.

### CODE REVIEW

**Review by cmccarthy1 — APPROVED** [review]

File: docs/faq.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+> 
+> Even if you are running Docker on macOS or Windows the version of kdb+ is 64-bit Linux, and the Docker image is Linux.
+
+
+## Can I restrict the access to my notebook using passwords?
+
+Yes, you can initialize a notebook which requires other processes to provide a valid *username:password* pair when trying to open a connection with the notebook.
+
+:point_right:
+[Restrict access](https://github.com/KxSystems/ml/tree/master/docs/notebooks.md#restrict-access-to-a-notebook-using-passwords)
```
Comment (jhanna-kx) [inline docs/faq.md:73]: broken link

File: docs/faq.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+:point_right:
+[Loading and saving scripts](https://github.com/KxSystems/ml/tree/master/docs/notebooks.md#loading-and-saving-code)
+
+
+## Can I mix Python and q code in the same notebook? 
+
+Yes, either with `p)` or `/%python`.
+
+:point_right:
+[Examples](https://github.com/KxSystems/ml/tree/master/docs/notebooks.md#python-and-inline-display)
```
Comment (jhanna-kx) [inline docs/faq.md:50]: broken link

**Comment by jhanna-kx** [comment]
some broken links in faq.md

**Comment by StephenTaylor-Kx** [comment]
code.kx.com/q/interfaces now links to this repo for documentation

**Comment by jhanna-kx** [comment]
@StephenTaylor-Kx were you able to fix the broken links I flagged above?

**Pull request closed without merging**